### PR TITLE
Issue #151 : Add support for PHP 8

### PR DESCRIPTION
**Pull request by Ciloe**
This PR is related to : 
- [x] Add Github Action due to unsuported PHP 8.0 on travis
- [x] Remove support on PHP 7.0 and 7.1 because phpunit 9 is the only version that supported PHP8 and is not compatible with old php versions
- [x] Correct deprecation on PHP8 for usort function
- [x] Update tests for compatibilities on phpunit 8/9

Signed-off-by: Jimmy ESCRICH <jescrich@ruedesvignerons.com>

### CODE REVIEW

**Comment by Ciloe** [comment]
All tests passing on Github Action https://github.com/Ciloe/twigcs/actions/runs/343278105

**Comment by jaylinski** [comment]
@OwlyCode Since PHP 8 is now released, it would be really nice to get this merged and tagged. :)

File: .github/workflows/ci-tests.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-versions }}
+          tools: phpunit
+
+      - name: Install dependencies
+        run: composer --prefer-source -n install
```
Comment (oallain) [inline .github/workflows/ci-tests.yaml:28]: Hello,

What is the purpose of using the `--prefer-source` here,  `--prefer-dist` would not be better?

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,7 @@
 language: php
 php:
-  - 7.0
-  - 7.1
   - 7.2
   - 7.3
+  - 7.4
```
Comment (oallain) [inline .travis.yml:4]: If we're using Github Action now, maybe it's time to remove Travis?

File: .github/workflows/ci-tests.yaml (hunk 1 above)
Comment (nicwortel) [reply .github/workflows/ci-tests.yaml:28]: I'm not @Ciloe, but I'm assuming it's a direct copy from the existing `.travis.yml`:

> ```
> install:
>   - composer --prefer-source -n install
> ```

Comment (oallain) [reply .github/workflows/ci-tests.yaml:28]: Ok, I see but, is it wise or not?

Comment (Ciloe) [reply .github/workflows/ci-tests.yaml:28]: Yes it's a simple copy of travis

File: .travis.yml (hunk 2 above)
Comment (Ciloe) [reply .travis.yml:4]: Yes, but I would like to see the transition before deleted it

**Review by nicwortel — APPROVED** [review]
The changes look good. I think changing Composer commands would be out of scope for this PR and should be handled separately.

**Comment by Chi-teck** [comment]
PHP 8 is now supported on Travis. If migrating to GitHub acions cannot be done quikly can we add support for PHP 8 without it?

File: .travis.yml (hunk 2 above)
Comment (Ciloe) [inline .travis.yml:4]:
```suggestion
  - 7.4
  - 8.0
```

**Review by oallain — APPROVED** [review]

**Comment by stof** [comment]
@Ciloe what are the plan regarding merging this ?

**Comment by Ciloe** [comment]
This PR will be merge soon. Normaly this week

**Pull request merged**